sdk/src/driftClient.ts

```diff
@@ -616,12 +614,12 @@ export class DriftClient {
 			}
 		} else {
 			const userAccounts =
-				(await this.getUserAccountsForAuthority(this.authority)) ?? [];
+				(await this.getUserAccountsForAuthority(this.wallet.publicKey)) ?? [];
 			let delegatedAccounts = [];
 
 			if (this.includeDelegates) {
 				delegatedAccounts =
-					(await this.getUserAccountsForDelegate(this.authority)) ?? [];
+					(await this.getUserAccountsForDelegate(this.wallet.publicKey)) ?? [];
```

Comment:
need to confirm it's the wallet key for this and the above - don't want to be accidentally pulling all users accounts or delegate accounts of a delegate authority

Reply:
think it being wallet.publicKey makes sense here